_inc/client/plans/product-selector.jsx

```diff
@@ -256,11 +256,14 @@ export default connect( state => {
 		multisite: isMultisite( state ),
 		products: getProducts( state ),
```

Comment:
I think in one of the earlier comments you mentioned this component got slower to load (due to the extra API endpoint)...

I think you can actually eliminate this `products` property from the component (and the `isFetchingProducts` property below) and just pass `siteProducts` in to `SingleProductBackup` instead -- this should work fine since the two have the same structure (and for Jetpack Backup, the exact same data).

In that case, you can probably also avoid calling `QueryProducts` on this page entirely, which would avoid having to hit that API endpoint at all.